mkfs.c

```diff
@@ -90,6 +90,7 @@ static int write_inode_store(int fd, struct superblock *sb)
     uint32_t first_data_block = 1 + le32toh(sb->info.nr_bfree_blocks) +
                                 le32toh(sb->info.nr_ifree_blocks) +
                                 le32toh(sb->info.nr_istore_blocks);
+    inode += 1;  // use inode 1 for root
```

Comment:
Improve the comment. Explain the reason. In addition, use C-style comments. i.e., `/* --- */` instead of `//`.

Reply:
update.